### PR TITLE
mednafen_pce_fast_libretro: a port of Mednafen PCE Fast to the libretro API

### PR DESCRIPTION
**Pull request by kwyxz**
This is a port of Mednafen PCE Fast to libretro, which is a faster version of Mednafen PCE. It sacrifices a bit of accuracy to increase speed. Mednafen is a multi-system emulator that uses forks of other emulators for some systems, and original code for others. The libretro cores use standalone systems from Mednafen.

### CODE REVIEW

File: games-emulation/mednafen_pce_fast_libretro/mednafen_pce_fast_libretro-0.9.38.7_20180220.recipe [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+and SuperGrafx emulator. This is one of the original emulators in Mednafen \
+that is not a fork from another."
+HOMEPAGE="https://mednafen.github.io/documentation/pce_fast.html"
+COPYRIGHT="2005-2018 Forgotten, the Mednafen team, the libretro team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="20105f7966951e062f6cc78cc4a3c932b55d8b3e"
+SOURCE_URI="https://github.com/libretro/beetle-pce-fast-libretro/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="044e629c56c8f8320ca61ee6e4bc7cc980590f68e3a8f1896ab6a2c17235cd66"
+SOURCE_FILENAME="libretro-mednafen-pce-fast-$portVersion-$srcGitRev.tar.gz"
```
Comment (fbrosson) [inline games-emulation/mednafen_pce_fast_libretro/mednafen_pce_fast_libretro-0.9.38.7_20180220.recipe]: Suggestion:
```diff
-SOURCE_FILENAME="libretro-mednafen-pce-fast-$portVersion-$srcGitRev.tar.gz"
+SOURCE_FILENAME="libretro-mednafen-pce-fast-${portVersion/_/-}-$srcGitRev.tar.gz"
```

File: games-emulation/mednafen_pce_fast_libretro/mednafen_pce_fast_libretro-0.9.38.7_20180220.recipe [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	sed -e "s/@DISPLAY_VERSION@/v${portVersion}/" \
```
Comment (fbrosson) [inline games-emulation/mednafen_pce_fast_libretro/mednafen_pce_fast_libretro-0.9.38.7_20180220.recipe]: Suggestion:
```diff
-	sed -e "s/@DISPLAY_VERSION@/v${portVersion}/" \
+	sed -e "s/@DISPLAY_VERSION@/v${portVersion/_/-}/" \
```

**Comment by kwyxz** [comment]
Uh yes we should. Sorry I forgot it and thank you for taking the time to fix it everywhere.

**Comment by fbrosson** [comment]
No problem :)
Yep, I came to the conclusion that it was what you meant, so I did the change as I knew you're in a quite different timezone.
BTW, it's really cool to have retroarch and all these libretro cores :)

**Comment by kwyxz** [comment]
Yeah I am in California. Thank *you* for your very patient help, not so many open source projects I worked with have been that helpful. There's more cores to come!

**Pull request merged**